cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -245,6 +245,18 @@ private HiveConf getHiveConf() {
       conf.set(HIVE_METASTORE_TOKEN_KEY, HiveAuthFactory.HS2_CLIENT_TOKEN);
     }
 
+    // workaround to allow CDAP explore to modify params of session conf
+    String whiteListAppend = conf.getVar(HiveConf.ConfVars.HIVE_AUTHORIZATION_SQL_STD_AUTH_CONFIG_WHITELIST_APPEND);
+    String paramsExploreModifies =
+      "spark.*|explore.*|mapreduce.*|hive.*|" + CommonConfigurationKeysPublic.HADOOP_SECURITY_AUTHENTICATION;
```

Comment:
Basically we are allowing everything here. It would be good to explicitly list the parameters that explore needs to modify in session conf. We can let the users set rest of the parameters, when they need it.

@anew What do you suggest?